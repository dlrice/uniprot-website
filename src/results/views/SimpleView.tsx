import React from 'react';

const SimpleView = (props: { termValue: string }) => {
  const { termValue } = props;
  return <div>{termValue}</div>;
};

export default SimpleView;
