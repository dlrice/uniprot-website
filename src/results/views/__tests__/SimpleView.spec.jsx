import React from 'react';
import { shallow, configure } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
import SimpleView from '../SimpleView';

configure({ adapter: new Adapter() });

describe('SimpleView component', () => {
  test('should render', () => {
    const wrapper = shallow(<SimpleView termValue="blah" />);
    expect(wrapper.debug()).toMatchSnapshot();
  });
});
