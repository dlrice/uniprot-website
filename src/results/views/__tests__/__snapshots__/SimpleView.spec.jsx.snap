// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SimpleView component should render 1`] = `
"<div>
  blah
</div>"
`;
